applications/serial_lte_modem/doc/slm_description.rst

```diff
@@ -1639,6 +1639,276 @@ Complete the following steps to test the functionality provided by the :ref:`SLM
       #XPING: average 0.650
       OK
 
+Testing GPS AT commands
+-----------------------
+
+Before you test the GPS AT commands, check the following configuration values:
+
+* :option:`CONFIG_SUPL_CLIENT_LIB` - must be enabled to use the :ref:`supl_client`
+* :option:`CONFIG_SLM_SUPL_SERVER` - use the default ("supl.google.com") or specify a different server
+* :option:`CONFIG_SLM_SUPL_PORT` - use the default (7276) or specify a different port
+
+Complete the following steps to test the functionality provided by the :ref:`SLM_AT_GPS`:
+
+1. Test standalone GPS:
+
+   a. Enable support for GNSS in the modem.
+
+      .. parsed-literal::
+         :class: highlight
+
+         **AT+CFUN=0**
+         OK
+
+         **AT%XSYSTEMMODE=0,0,1,0**
+         OK
+
+   .. modem_config_start
+
+   b. Configure the Low-Noise Amplifier (LNA).
+      The required commands depend on the version of the nRF9160 DK.
+
+      For nRF9160 DK versions up to v0.9.0, configure the MAGPIO pin:
+
+      .. parsed-literal::
+         :class: highlight
+
+         **AT%XMAGPIO=1,0,0,1,1,1574,1577**
+         OK
+
+      For nRF9160 DK versions starting with v0.10.x, configure the COEX0 pin:
+
+      .. parsed-literal::
+         :class: highlight
+
+         **AT%XCOEX0=1,1,1570,1580**
+         OK
+
```

Comment:
For DK v0.10.0~, GPS frequency has been expanded from 1570-1580 to 1565-1586. So the AT command should be "AT%XCOEX0=1,1,1565,1586" now.

Reply:
Thanks, fixed.